script.js

```diff
@@ -0,0 +1,24 @@
+const currencyRate = {
+  timestamp: 1620386223,
+  base: "DKK",
+  rates: {
```

Comment:
ok, I did not get it correctly first but 

` Create a form to insert a brand new currency rate from a to b.` means that you need to create a new form which you use to insert the rates which you can use afterwards